node.js/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -3,13 +3,73 @@ const router = express.Router();
 
 const meals = require("./../data/meals.json");
 
-router.get("/", async (request, response) => {
+router.get("/:id", async (request, response) => {
+  console.log(meals);
   try {
-    console.log(meals);
-    console.log("in /api/meals");
+    const mealId = parseInt(request.params.id);
+
+    if (isNaN(mealId)) {
+      response.status(404).json({ error: "Id must be integers." })
```

Comment:
400 - bad request is better than 404...you can use 404 if you have a valid id but no meal associated to it.